be/src/types/bitmap_value.cpp

```diff
@@ -587,10 +587,10 @@ int64_t BitmapValue::cardinality() const {
     return 0;
 }
 
-uint64_t BitmapValue::max() const {
+std::optional<uint64_t> BitmapValue::max() const {
```

Comment:
we should return NULL when  _set->size() == 0

Reply:
done